lib/output.js

```diff
@@ -85,7 +85,8 @@ module.exports.circular = function (spinner, res, circular, opts) {
 				if (idx) {
 					process.stdout.write(chalk.dim(' > '));
 				}
-				process.stdout.write(chalk.cyan.bold(module));
+				// Format to make vscode ref link works (#291)
+				process.stdout.write(chalk.cyan.bold(module.split('').splice(4).join('')));
```

Comment:
I'm not sure what is the purpose of `.split('').splice(4).join('')` seems it's same as `.substring(4)`?
Also, is this compatible with other folders? I'm not sure where the `..\..\` come from. Could you provide more details in #291 ?